external/buildscripts/build.pl

```diff
@@ -1509,6 +1512,20 @@
 			system("rm -rf $distdirlibmono/gac/nunit*");
 		}
 
+		if ($buildMSBuild)
+		{
+			#Our classlibrary build is missing String.format, so discard our changes before building msbuild
+			my $currentPath = $ENV{PATH};
+			$ENV{PATH} = $startPath;

```

Comment:
You use a system Mono to build then?

Reply:
I still need to do more digging, but I suspect our ENV is being polluted by .NET standard 2.0...maybe if we build the full 4_6 profile last we'll be okay, or explicitly set that in the env